backend/src/main/java/student/aschm22s/hbrsiCalGenerator/calenderExport/contoller/CalenderExportController.java

```diff
@@ -30,29 +29,29 @@ public CalenderExportController(CalenderExportService calenderExportService, Ema
             method = {RequestMethod.POST},
             produces = "text/calender"
     )
-    public String getCalenderOverEmail(@RequestBody VeranstaltungsIdsAndEmail veranstaltungsIdsAndEmail) {
-        if (Arrays.stream(blacklistedEMails).anyMatch(x -> veranstaltungsIdsAndEmail.getEmail().equals(x))) {
+    public String getCalenderOverEmail(@RequestBody VeranstaltungsIdsAndEmailDAO veranstaltungsIdsAndEmailDAO) {
+        if (Arrays.stream(blacklistedEMails).anyMatch(x -> veranstaltungsIdsAndEmailDAO.getEmail().equals(x))) {
```

Comment:
DAO -> Data Access Object
DTO -> Data Transfer Object

Ich denke du meinst Zweiteres, denn ein DAO implementiert in der Regel die Datenbankschicht ;)

Reply:
Ahh, ja genau das meinte ich^^